packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -19,6 +20,13 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
   const state = decodeOAuthState(req);
 
   if (typeof code !== "string") {
+    if (state?.onErrorReturnTo) {
+      res.redirect(
+        getSafeRedirectUrl(state.onErrorReturnTo) ??
+          getSafeRedirectUrl(state?.returnTo) ??
+          `${CAL_URL}/apps/installed`
```

Comment:
Fallback to the original returnTo URL if the error one is not present.

Reply:
@joeauyeung I think we need to use WEBAPP_URL here?